packages/lib/posthogServer.ts

```diff
@@ -34,3 +35,33 @@ export const capturePosthogEnvironmentEvent = async (
     console.error("error sending posthog event:", error);
   }
 };
+
+export const sendPlanLimitsReachedEventToPosthogWeekly = (
+  environmentId: string,
+  meta: {
```

Comment:
would `organizationBilling` or `billing` make more sense here instead of `meta`?